amy/extforms/forms.py

```diff
@@ -192,11 +195,32 @@ def create_consent_field(self, term: Term) -> forms.ChoiceField:
         )
         return field
 
+    @feature_flag_enabled("ENFORCE_MEMBER_CODES")
+    def validate_member_code(self, request: HttpRequest) -> dict:
+        errors = dict()
+        member_code = self.cleaned_data.get("member_code", "")
+
+        if not member_code:
+            return
```

Comment:
Alternatively this could return empty dict.

Reply:
Actually, if the feature flag is disabled, then this function would return `None`, so I'd still keep `None` here and just correct the return type annotation in L199.